### ISSUE TITLE
Polish Translation

### ISSUE THREAD

**Issue by strusik01** [fmasa/wfrp-master#380]
Hello, I'm wondering if the app will be translated any time soon? I can see some languages covered, polish included which is the one I'm most interested with, although it's not implemented yet.

**Comment by fmasa** [comment]
I just [enabled](https://github.com/fmasa/wfrp-master/pull/390) the Polish in the language picker. You can enroll into testing track to get version of the app built from `master` using this link https://play.google.com/apps/testing/cz.frantisekmasa.dnd to test it.

cc @oswida

**Comment by fmasa** [comment]
If you are satisfied with the translation, this can be released with other changes in 3.21

**Comment by strusik01** [comment]
That's great to hear :D

I've installed the dev-207278c, but it has no option for Polish yet. Is it the build you've just enrolled, or is it not live yet?

**Comment by fmasa** [comment]
It's available in `dev-0eeccd4` which I see as the latest in the Google Play. It should be available as update in few hours (at latest). The 207278c is quite old :slightly_smiling_face: 

Let me know if it does not appear for you :slightly_smiling_face:

**Comment by strusik01** [comment]
It just appeared, with Polish setting available.

Although something went wrong, changing the language in settings doesn't change it in app at all now. 
Its the same for all languages, its English only

**Comment by fmasa** [comment]
That is odd, it works for me.

Is it possible that you are accidentally changing the Party language rather than app language?
If not, can you try restarting the app after the language change?

**Comment by strusik01** [comment]
Solved the issue by reinstalling the app, I guess my installation broke. It works now ^^

I'll begin translating the Compendium now and playtest it for a few days, I'll let you know if something goes wrong. 
Thank you very much and Happy New Year ;D

**Comment by fmasa** [comment]
Glad it works now (though it's still odd :slightly_smiling_face:).
If you could test the Talent/Trait translation as well, that would be great! (See [Effects](https://github.com/fmasa/wfrp-master/wiki/Automation#effects) for that).

Happy New Year to you too :slightly_smiling_face:

**Comment by strusik01** [comment]
I've found some issues, I can create bugs if you wish, but I'll specify them here for now:

1. Polish word for "Base" is cut (too long)
**Location**: Character creation -> Attributes
**Possible solution**: Smaller font OR changing translation to "początk." so it would fit
**Screenshot**: 
![1 -Creating Character - Characteristics](https://github.com/user-attachments/assets/56354a98-db70-456c-9b6c-649976fbde31)

2. Polish words for "Basic Info" and "Point Pools" tabs overlap (too long)
**Location**: Character creation -> Attributes
**Possible solution**: Wrapping text for "Basic Info" (alias "Informacje Początkowe") tab
**Screenshot**: 
![2 - Creating Character - Tabs](https://github.com/user-attachments/assets/a082e1ec-cf3b-49fa-92e8-5f89a39bd2ee)

3. Polish words for "Fate" and "Resolve" wrapped in a weird way ;)
**Location**: Character sheet -> Attributes tab
**Possible solution**: Not sure for this one. Smaller font maybe, but it could be way too small for other languages. Maybe spreading the point pools to the sides would make enough space?
**Screenshot**: 
![3 - Character Sheet - Points](https://github.com/user-attachments/assets/b7647543-9c72-46f3-bfa8-4929bb632522)

4. Journal references links for Symptoms, Weapon/Armour/Item qualities, flaws and groups. 
Links look for base english parent folders, though I saw them translated on Weblate. Its less of an issue for Symptoms, as I can still name them in polish, but polish qualities flaws and groups look for english names in Journal.
**Location**: Disease or Weapon/Armour/Item trapping with added qualities/flaws/group
**Screenshots**: 
![4 - Journal Translation on Weblate](https://github.com/user-attachments/assets/6b21d606-92cb-4ed8-8bdf-504be6da27ad)
![4 - Weapon Armour Item - Qualities Flaws Groups reference linking](https://github.com/user-attachments/assets/4b54ed30-168d-414f-85de-d7cccd128e0e)
![4 - Weapon - Group reference link](https://github.com/user-attachments/assets/9850b848-e4c3-4f17-96c7-ff316a8738ec)
![4 - Diseases - Symptom linking](https://github.com/user-attachments/assets/75c4062d-5697-4456-8558-a6a34fe7dfa9)

**Comment by fmasa** [comment]
Rather than bugs, you can change the translation in Weblate and I will
merge it. 🙂

**Comment by strusik01** [comment]
I can make shorts for some, but what about Journal? Links try to find English journal entries, despite being translated to polish in Weblate. Is there a way to change parent folders for polish version?

**Comment by fmasa** [comment]
@strusik01 I'll take a look :thinking: There is likely a bug.

**Comment by strusik01** [comment]
I've checked Talents/Traits, unfortunately they're not calculated at all if translated. 
For example Savvy, if translated to Błyskotliwość, doesn't add up to Character's Intelligence.
As for Size trait it's the same, its not recognized if translated to Rozmiar, various sizes like Monstrous/Monstrualny probably aren't too.

**Comment by fmasa** [comment]
@strusik01 I'll take a look. I quickly tested German translation and it works correctly, so there is likely a problem with configuration of specific locales.

**Comment by fmasa** [comment]
@strusik01 fixed :slightly_smiling_face: 

(I closed the issue by accident :slightly_smiling_face:)

**Comment by strusik01** [comment]
It's mostly fixed, talents and traits work correctly. 

As of the Journal, while Consumer's Guide and Conditions have translated and working well parent folders and links, the Disease and Infection ones don't work. Error message still states that its not translated, although now it does not work for neither English or Polish path (if language set to Polish).

**Comment by fmasa** [comment]
@strusik01 Good catch! The click through works, but the toast message used untranslated string. Fixed in https://github.com/fmasa/wfrp-master/pull/398

**Comment by strusik01** [comment]
All good now, everything I've checked works. Polish is ready for release, thank you very much 😄

**Comment by fmasa** [comment]
Great! I tagged [3.21.0](https://github.com/fmasa/wfrp-master/releases/tag/3.21.0) just now :slightly_smiling_face:
